pkgs/by-name/re/refine/package.nix

```
@@ -0,0 +1,100 @@
+{
+  lib,
+  fetchFromGitLab,
+  nix-update-script,
+  blueprint-compiler,
+  desktop-file-utils,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+  python3,
+  libxml2,
+  python3Packages,
+  libportal,
+  libportal-gtk4,
+  appstream,
+  gtk4,
+  glib,
+}:
+
+let
+  libadwaita' = libadwaita.overrideAttrs (oldAttrs: {
+    version = "1.6.2-unstable-2025-01-02";
+    src = oldAttrs.src.override {
+      rev = "f5f0e7ce69405846a8f8bdad11cef2e2a7e99010";
+      hash = "sha256-n5RbGHtt2g627T/Tg8m3PjYIl9wfYTIcrplq1pdKAXk=";
+    };
```

I'm assuming the app itself works fine here, but this build of libadwaita has some issues with tests; doesn't seem to affect anything though

```suggestion
    };

    # `test-application-window` is flaky on aarch64-linux 
    doCheck = false;
```